sdk/test/CSMTest/Tests/UnitTests/CSMUtilitiesTests/AWSSDK.CSMUtilitiesTests.NetFramework/AWSSDK.CSMUtilitiesTests.NetFramework.csproj

```diff
@@ -1,13 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFramework>net462</TargetFramework>
     <DefineConstants>$(DefineConstants);DEBUG;TRACE;;BCL;BCL45;ASYNC_AWAIT;LOCAL_FILE</DefineConstants>
     <DebugType>portable</DebugType>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <AssemblyName>AWSSDK.CSMUtilitiesTests.Net45</AssemblyName>
-    <PackageId>AWSSDK.CSMUtilitiesTests.Net45</PackageId>
+    <AssemblyName>AWSSDK.CSMUtilitiesTests.NetFramework</AssemblyName>
+    <PackageId>AWSSDK.CSMUtilitiesTests.NetFramework</PackageId>
 
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
After I changed the target framework, I saw this error (which went away after adding this attribute):
```
CS0579	Duplicate 'System.Reflection.AssemblyInformationalVersionAttribute' attribute	AWSSDK.CSMUtilitiesTests.NetFramework
```